cg/services/order_validation_service/validators/data/rules.py

```diff
@@ -73,6 +75,15 @@ def validate_application_not_archived(
     return errors
 
 
+def validate_gene_panels_unique(order: TomteOrder, **kwargs) -> list[CaseError]:
+    errors: list[CaseError] = []
+    for case in order.cases:
+        if not len(set(case.panels)) == len(case.panels):
```

Comment:
Extract conditional to function with meaningful name

Reply:
Extracted into helper function